ledger/error/src/main/scala/com/daml/error/generator/ErrorCodeDocumentationGenerator.scala

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.error.generator
+
+import java.lang.reflect.Modifier
+import com.daml.error.ErrorCode
+import com.daml.error.generator.ErrorCodeDocumentationGenerator.{
+  explanationTypeName,
+  resolutionTypeName,
+}
+import com.daml.error.{Explanation, Resolution}
+import org.reflections.Reflections
+
+import scala.reflect.runtime.{universe => ru}
```

Comment:
Why do we need this import alias? The code would be easier to read if you directly imported `Annotation`.